packages/apps/plugins/plugin-debug/src/components/Wireframe.tsx

```diff
@@ -5,12 +5,18 @@
 import React from 'react';
 import { useResizeDetector } from 'react-resize-detector';
 
+import { createAttendableAttributes } from '@dxos/react-ui-attention';
 import { mx } from '@dxos/react-ui-theme';
 
+// TODO(burdon): Make focusable and attendable with input.
 export const Wireframe = ({ label, data, className }: { label?: string; data?: any; className?: string }) => {
+  const attendableAttrs = createAttendableAttributes(data);
+  // const x = useHasAttention(attendableAttrs);
+  // console.log(x);
```

Comment:
👀

Reply:
TODO was to make this attendable so that we have an obvious check for arrow navigation across the app.